flink-ml-lib/src/main/java/org/apache/flink/ml/classification/knn/FastDistanceVectorData.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.ml.classification.knn;
+
+import org.apache.flink.ml.linalg.DenseVector;
+import org.apache.flink.util.Preconditions;
+
+import java.io.Serializable;
+
+/** Save the data for calculating distance fast. The FastDistanceMatrixData */
+public class FastDistanceVectorData implements Serializable, Cloneable {
```

Comment:
Why does this class needs to be implements Serializable and Cloneable?

And if this class is supposed to be serialized/deserialized, how do we ensure that its serialization/deserialization logic would re-use the serialization/deserialization logic of `DenseVector`?

Reply:
done